powerlevel9k.zsh-theme

```diff
@@ -1298,6 +1298,8 @@ powerlevel9k_prepare_prompts() {
   # Reset start time
   _P9K_TIMER_START=0xFFFFFFFF
 
+  setopt PROMPT_SUBST
+
   if [[ "$POWERLEVEL9K_PROMPT_ON_NEWLINE" == true ]]; then
     PROMPT='$(print_icon 'MULTILINE_FIRST_PROMPT_PREFIX')%f%b%k$(build_left_prompt)
```

Comment:
Also, unrelated to this PR, but I'm fairly sure this line is broken (my fault). It would probably be better to use double quotes and escape the `$(build_left_prompt)`